templates/blazor-base/csharp/default/GettingStarted.txt

```diff
@@ -1,12 +1,25 @@
 Welcome to the Teams Toolkit! Here are some tips to get started building your app.
 
+Prerequisites
```

Comment:
To debug a Bot capability in a Teams app, you'll need to setup [ngrok](https://ngrok.com/). ngrok is used to help forward external messages from Azure Bot Framework to your locally running machine.

After installing ngrok, use a Command Prompt to run this command: ngrok http 2544